universal-login-relayer/test/helpers/WalletCreator.ts

```diff
@@ -0,0 +1,69 @@
+import fetch from 'node-fetch';
+import {Wallet, Contract, utils} from 'ethers';
+import {http, HttpFunction, PublicRelayerConfig, createKeyPair, calculateInitializeSignature, computeContractAddress, TEST_GAS_PRICE} from '@universal-login/commons';
+import {encodeInitializeWithRefundData} from '@universal-login/contracts';
+import ProxyCounterfactualFactory from '@universal-login/contracts/build/ProxyCounterfactualFactory.json';
+import ENSService from '../../lib/integration/ethereum/ensService';
+
+export class WalletCreator {
```

Comment:
Is it used anywhere? We should not commit dead code to repository.

Reply:
There is queued (on my computer) PR that I use WalletCreator in DevelopmentRelayer and TokenGrantingRelayer tests.